phpstan.neon

```diff
@@ -13,8 +13,8 @@ parameters:
     - vendor/php-stubs/wordpress-stubs/wordpress-stubs.php
     - vendor/php-stubs/wordpress-tests-stubs/wordpress-tests-stubs.php
   type_coverage:
-      return_type: 92
-      param_type: 65
+      return_type: 94
+      param_type: 72
       property_type: 0 # We can't use property types until we support PHP 7.4.
```

Comment:
Aside: we support 7.4; should this be changed?

Reply:
Good question!

Property types were introduced in PHP 7.4. We have PHP 7.2 set as the minimum for the plugin, and internally we're still using PHP 7.1 syntax.

If I try to introduce this somewhere (e.g. `protected array $metadata = array();`), I get complains from IntelliSense (`PHP 7.4+ syntax found. Targeting PHP 7.1.0.`) and PHPCS (`Typed properties are not supported in PHP 7.3 or earlier`).

So I think the comment is valid, it just needs an improved phrasing. I'll add this into my todo list.

Reply:
Addressed in https://github.com/Parsely/wp-parsely/pull/1752